DEVX_ROADMAP.md

```diff
@@ -1,51 +1,39 @@
-* Support for passing strings and object ID encodings to entry functions
-* This frequently requested feature allows passing a vector of objects to entry functions which will open up a variety of flexible use cases *wrt* payments and coin management.
-* A collection of cryptographic libraries are exposed as Move native functions to use on-chain. They can be found at:
-  * [crypto.move](https://github.com/MystenLabs/sui/blob/4ed29873e35cad9d709dbd9370e17d261446e873/crates/sui-framework/sources/crypto/elliptic_curve.move): This contains various utility methods for on-chain signature verification. It also provides a method to recover Secp256k1 signature to its public key, bulletproof verification, and Pedersen commitments. See this work in action in the Cryptography (math) [example](https://github.com/MystenLabs/sui/tree/main/sui_programmability/examples/math).
-  * [elliptic_curve.move](https://github.com/MystenLabs/sui/tree/main/sui_programmability/examples/math): This introduces a RistrettoPoint struct, which represents an elliptic curve point on the Ristretto255 sub-group of curve25519. Currently, we support the addition and subtraction of elements in the Ristretto255 sub-group, and operations of scalars within the field of curve25519.
-  * HKDF, AES, ECDH
+
+* Intent signing support which will include an intent struct to be serialized and signed in addition to the transaction data [[Issues](https://github.com/MystenLabs/fastcrypto/issues/26)].
+* [Rust SDK] Removing reliance on String and Json Values in Rust Transaction Builder, making it more Rust friendly
+* Adding denomination conversion functionality, which converts SUI to MIST.
+* [TypeScript] Adding support for deserializing transactions.
+
+## Sui Move
+
+* Crypto.move will be split into its individual modules organized by crypto primitives [[PR](https://github.com/MystenLabs/sui/pull/4653)].
+* Supporting dynamic child access requires additional time due to newer complexities discovered and we are actively working on enabling it on devNet ASAP. 
+* Support for passing mutable coins instead of owned.
 
 ## Sui CLI
-* Ability to use Sui CLI on full nodes ([issue](https://github.com/MystenLabs/sui/pull/4404))
-* Generate keypair now requires signature scheme flag
-  * `sui keytool generate ed25519` or `sui keytool generate secp256k1`
-  * `sui client new-address ed25519` or  `sui client new-address secp256k1`
-* Key derivation for Ed25519 following SLIP-0010
+* Intent signing support: Same as in SDK. Also, making the Sui keystore ChainID aware as well as password protected.
+
+## Faucet
+* As a precursor to integrating Faucet into Sui CLI/SDKs/Wallet, rate-limiting will be enforced on the faucet server to 5 requests per minute and no more than 20 requests in 24 hours, originating from the same ip address.
```

Comment:
Should we link to Chris' write up here for folks looking for further information? https://www.notion.so/mystenlabs/RFC-Faucet-Improvement-Plan-f706cd7411c046bc8fca3f1a7a8b559b

Reply:
We don't have an RFC in github for this yet.